### PR TITLE
publish storybook master only on version change

### PR DESCRIPTION
**Pull request by clxandstuff**
Publish on master will not upload new files until new version in package json.

### CODE REVIEW

**Review by coderitual — APPROVED** [review]

File: scripts/test-s3.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  },
+});
+
+client.s3.headObject(
+  {
+    Bucket: 'style-guide.brainly.com',
+    Key: `index.html`,
+  },
+  function(notExist) {
+    console.log(notExist);
```
Comment (coderitual) [inline scripts/test-s3.js:17]: maybe more descriptive?

File: scripts/build-storybook.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+});
+
+client.s3.getObject(
+  {
+    Bucket: 'style-guide.brainly.com',
+    Key: `.sg-version`,
+  },
+  function(err, data) {
+    if (!data || (data && data.toString('utf-8') !== version)) {
```
Comment (coderitual) [inline scripts/build-storybook.js:18]: maybe some console info if it's not met?

Comment (clxandstuff) [reply scripts/build-storybook.js:18]: Good idea, thanks

File: scripts/test-s3.js (hunk 1 above)
Comment (clxandstuff) [reply scripts/test-s3.js:17]: This is actually test file. I will remove it :) Thanks for catching it

**Pull request merged**